app/importers/scenario_import.rb

```diff
@@ -54,6 +54,9 @@ def import_confirmed?
     do_import == "1"
   end
 
+  def import!(options = {})
```

Comment:
it was the start of a refactor to have import check validity, the same way the ActiveRecord#save and #save! work. I've backed away from that change for the moment, but might go back to it